docs/user_manual.md

```diff
@@ -141,6 +141,7 @@ the available configuration keys with an explanation of each one.
   },
   "id" : "myservice:0.5:3539b7bc2235d53f79e6e8511942bbeaa8816265",
   "image" : "myregistry:80/janedoe/myservice:0.5-98c6ff4",
+  "hostname": "myhost",
```

Comment:
Can you also add "hostname: The hostname to pass to the container" to the explanation below starting at line 174? Make sure to put it in the same order as the JSON here.